src/redist/targets/GenerateBundledVersions.targets

```diff
@@ -211,8 +215,8 @@
         Condition="'$(DotNetBuildFromSource)' == 'true'"
         Include="$(ProductMonikerRid)" />
 
-      <MonoRuntimePackRids Include="
-          @(NetCoreRuntimePackRids);
+      <Net60MonoRuntimePackRids Include="
+          @(Net70RuntimePackRids);
```

Comment:
Is this correct?  It seems backwards that .NET 6 RIDS would be a superset of .NET 7 RIDS.